api/src/main/java/marquez/service/models/Job.java

```diff
@@ -56,7 +59,10 @@ public Job(
       @Nullable final String description,
       @Nullable final Run latestRun,
       @Nullable final ImmutableMap<String, Object> facets,
-      @Nullable UUID currentVersion) {
+      @Nullable UUID currentVersion,
+      @Nullable final String jobType,
+      @Nullable final String processingType,
+      @Nullable final String integration) {
     this.id = id;
     this.type = type;
```

Comment:
Note, `type` here can ether be `BATCH` or `STREAMING` and can drop `jobType` (see [`common/models/JobType.java`](https://github.com/MarquezProject/marquez/blob/main/api/src/main/java/marquez/common/models/JobType.java)).

Reply:
As [mentioned](https://github.com/MarquezProject/marquez/pull/2682#discussion_r1415230257) here, it's totally OK to remap OpenLineage properties to the Marquez specific model. I'd favor mapping `processingType` to the `JobType` defined in Marquez and document it in the API / docs.